apps/examples/i2schar/i2schar_receiver.c

```diff
@@ -80,27 +80,32 @@
  *   This is the entry point for the receiver thread.
  *
  ****************************************************************************/
+FAR struct ap_buffer_s *g_apb[CONFIG_EXAMPLES_I2SCHAR_TXBUFFERS];
```

Comment:
How is this array protected between read & write?

Reply:
First, this array is filled up by RX code, then it is used by TX code within the same thread.